### ISSUE TITLE
`verify` segfault on aarch64 linux

### ISSUE THREAD

**Issue by syoyo** [lighttransport/embree-aarch64#16]
OS: Ubuntu 18.04.3 Linux
CPU: ARM a72(aarch64)
Build config : Use `scripts/bootstrap-aarch64-linux.sh`
Branch : `master` v3.6.1

```
./verify 
Cylinder test 3 failed: cylinder = Cylinder { p0 = (0, 0, 0), p1 = (1, 0, 0), r = 1}, ray = { 
  org = (0, 0, 0)
  dir = (1, 0, 0)
  near = 0
  far = inf
  time = 0
  mask = -1
  id = 0
  flags = 0
  Ng = (-3.60566e+18, 1.77965e-43, -1.69402e-24)  u = 1.77965e-43
  v = -8.17928e-33
  primID = 85
  geomID = 4294967295
  instID = 0
}, hit = 1, t = [-295.603; 295.603]
Cylinder test 4 failed: cylinder = Cylinder { p0 = (0, 0, 0), p1 = (1, 0, 0), r = 1}, ray = { 
  org = (0, 0, 0)
  dir = (-1, 0, 0)
  near = 0
  far = inf
  time = 0
  mask = -1
  id = 0
  flags = 0
  Ng = (-3.60566e+18, 1.77965e-43, -1.69402e-24)  u = 1.77965e-43
  v = -8.17928e-33
  primID = 85
  geomID = 4294967295
  instID = 0
}, hit = 1, t = [-295.603; 295.603]
verify: /mnt/data/work/embree-aarch64/kernels/common/device.cpp:67: embree::Device::Device(const char*): Assertion `isa::Cylinder::verify()' failed.
Aborted (core dumped)
```

**Comment by syoyo** [comment]
The reason was rcp_length(rsqrt) used in `Cylinder::verify()` does not use correct NEON code path for non-iOS build

https://github.com/lighttransport/embree-aarch64/blob/master/common/math/vec3fa.h#L192

resulting insufficient or incorrect result(e.g. rcp_length(dot((1, 0, 0), (1, 0, 0))) gives slightly less value than 1(0.999994))

**Comment by syoyo** [comment]
Fixed in `neon-fix` branch(`aarch64-v3.8.0`)

**Comment by syoyo** [comment]
Now fix is included in `master` https://github.com/lighttransport/embree-aarch64/commit/443361810a7955d1e62af333ddb653eb3a949ad3
